pages/admin/request/create-new.tsx

```diff
@@ -0,0 +1,845 @@
+      requiresWiderParkingSpaceReason: null,
+      otherRequiresWiderParkingSpaceReason: null,
+    });
+
+    setPaymentDetails({
+      paymentMethod: null,
+      donationAmount: '',
+      shippingAddressSameAsHomeAddress: false,
+      shippingFullName: '',
+      shippingAddressLine1: '',
+      shippingAddressLine2: '',
+      shippingCity: '',
+      shippingProvince: 'BC',
+      shippingCountry: '',
+      shippingPostalCode: '',
+      billingAddressSameAsHomeAddress: false,
+      billingFullName: '',
+      billingAddressLine1: '',
+      billingAddressLine2: '',
+      billingCity: '',
+      billingProvince: 'BC',
+      billingCountry: '',
+      billingPostalCode: '',
+    });
+  };
+
+  /**
+   * Get information about applicant to pre-populate form
+   */
+  const [getApplicant, { loading: getApplicantLoading }] = useLazyQuery<
+    GetApplicantNewRequestInfoResponse,
+    GetApplicantNewRequestInfoRequest
+  >(GET_APPLICANT_NEW_REQUEST_INFO_QUERY, {
+    fetchPolicy: 'network-only',
+    onCompleted: data => {
+      if (data) {
+        const {
+          firstName,
+          middleName,
+          lastName,
+          dateOfBirth,
+          gender,
+          otherGender,
+          phone,
+          email,
+          receiveEmailUpdates,
+          addressLine1,
+          addressLine2,
+          city,
+          postalCode,
+          medicalInformation: { physician },
+          guardian,
+        } = data.applicant;
+
+        // set permitHolderInformation
+        setPermitHolderInformation({
+          firstName,
+          middleName,
+          lastName,
+          dateOfBirth: formatDateYYYYMMDD(new Date(dateOfBirth)),
+          gender,
+          otherGender,
+          email,
+          phone,
+          receiveEmailUpdates,
+          addressLine1,
+          addressLine2,
+          city,
+          postalCode,
+        });
+
+        // set doctorInformation
+        setDoctorInformation({
+          firstName: physician.firstName,
+          lastName: physician.lastName,
+          mspNumber: physician.mspNumber,
+          phone: physician.phone,
+          addressLine1: physician.addressLine1,
+          addressLine2: physician.addressLine2,
+          city: physician.city,
+          postalCode: physician.postalCode,
+        });
+
+        // set guardianInformation
+        if (guardian) {
+          setGuardianInformation({
+            omitGuardianPoa: false,
+            firstName: guardian.firstName,
+            middleName: guardian.middleName,
+            lastName: guardian.lastName,
+            phone: guardian.phone,
+            relationship: guardian.relationship,
+            addressLine1: guardian.addressLine1,
+            addressLine2: guardian.addressLine2,
+            city: guardian.city,
+            postalCode: guardian.postalCode,
+            poaFormUrl: '', //TODO
+          });
+        } else {
+          setGuardianInformation({
+            omitGuardianPoa: false,
```

Comment:
```suggestion
            omitGuardianPoa: true,
```

Reply:
Just wondering, why do we want the default to be for applicants to not have a guardian? Do most permit holders not have guardians? When this is set to true it hides the form so it might be easier to accidentally miss if we set it to true initially

Reply:
True, my bad, that's right